packages/lib/reducer.ts

```diff
@@ -78,6 +78,7 @@ export interface State {
 	hasDisabledSyncItems: boolean;
 	hasDisabledEncryptionItems: boolean;
 	customCss: string;
+	has_legacy_templates: boolean;
```

Comment:
Come on - look at the code around a bit. There's like a hundred variables in camel case here, and you set yours with underscores...

Reply:
done